1.26/README.md

```diff
@@ -0,0 +1,204 @@
+as a base image for other applications based on nginx 1.26 web server.
+Nginx server image can be extended using Openshift's `Source` build feature.
+
+
+Usage in OpenShift
+------------------
+In this example, we assume that you are using the `ubi8/nginx-126` image, available through the `nginx:1.26` imagestream tag in Openshift.
+To build a simple [test-app](https://github.com/sclorg/nginx-container/tree/master/examples/1.26/test-app) application in Openshift:
+
+```
+oc new-app nginx:1.26~https://github.com/sclorg/nginx-container.git --context-dir=1.26/test/test-app/
+```
+
+To access the application:
+```
+$ oc get pods
+$ oc exec <pod> -- curl 127.0.0.1:8080
+```
+
+
+Source-to-Image framework and scripts
+-------------------------------------
+This image supports the [Source-to-Image](https://docs.openshift.com/container-platform/4.14/openshift_images/create-images.html#images-create-s2i_create-images)
+(S2I) strategy in OpenShift. The Source-to-Image is an OpenShift framework
+which makes it easy to write images that take application source code as
+an input, use a builder image like this Nginx container image, and produce
+a new image that runs the assembled application as an output.
+
+In case of Nginx container image, the application source code is typically
+either static HTML pages or configuration files.
+
+To support the Source-to-Image framework, important scripts are included in the builder image:
+
+* The `/usr/libexec/s2i/run` script is set as the default command in the resulting container image (the new image with the application artifacts).
+
+* The `/usr/libexec/s2i/assemble` script inside the image is run to produce a new image with the application artifacts. The script takes sources of a given application (HTML pages), Nginx configuration files, and places them into appropriate directories inside the image. The structure of nginx-app can look like this:
+
+**`./nginx.conf`**--
+       The main nginx configuration file
+
+**`./nginx-cfg/*.conf`**
+       Should contain all nginx configuration we want to include into image
+
+**`./nginx-default-cfg/*.conf`**
+       Contains any nginx config snippets to include in the default server block
+
+**`./nginx-start/*.sh`**
+       Contains shell scripts that are sourced right before nginx is launched
+
+**`./nginx-perl/*.pm`**
+       Contains perl modules to be use by `perl_modules` and `perl_require` directives
+
+**`./`**
+       Should contain nginx application source code
+
+
+Build an application using a Dockerfile
+---------------------------------------
+Compared to the Source-to-Image strategy, using a Dockerfile is a more
+flexible way to build an Nginx container image with an application.
+Use a Dockerfile when Source-to-Image is not sufficiently flexible for you or
+when you build the image outside of the OpenShift environment.
+
+To use the Nginx image in a Dockerfile, follow these steps:
+
+#### 1. Pull a base builder image to build on
+
+podman pull ubi8/nginx-126
```

Comment:
E.g. See here: https://github.com/sclorg/s2i-perl-container/blob/master/5.38/README.md. It does not make sense to fix README.md twice. The first time when we add a new version and the second time, if the version is supported, update to proper `podman pull` commands.

It does not make sense to have their old version, because we will forget it.

E.g. https://github.com/sclorg/postgresql-container/blob/master/16/root/usr/share/container-scripts/postgresql/README.md contains wrong version. My opinion is to use not currently supported version in README.md files but correct version.